### PR TITLE
Security: rename package to project

### PR DESCRIPTION
**Pull request by glaubinix**

### CODE REVIEW

File: docs/security-monitoring.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
 # Security Monitoring
 
-Private Packagist Security Monitoring searches the dependencies of your private packages for known security vulnerabilities.
-Packages are analyzed every time we find a new commit in your package and when a new vulnerability is published in one of the databases.
+Private Packagist Security Monitoring searches the dependencies of your projects (private packages with a composer.lock) for known security vulnerabilities.
+Projects are analyzed every time we find a new commit in your project and when a new vulnerability is published in one of the databases.
```
Comment (naderman) [inline docs/security-monitoring.md:4]:
```suggestion
Private Packagist Security Monitoring searches the dependencies of your projects for known security vulnerabilities.
Projects, which are Composer packages with a composer.lock file, are analyzed every time you push a new commit to your project and when a new vulnerability is published in one of the databases.
```

File: docs/security-monitoring.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
-The following databases are used to analyze your packages:
+The following databases are used to analyze your projects:
 * [FriendsOfPHP/security-advisories](https://github.com/FriendsOfPHP/security-advisories)
 
 <div class="row column">
     <div class="callout warning">
-        <p>Security monitoring is available for packages in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
+        <p>Security monitoring is available for projects in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
```
Comment (naderman) [inline docs/security-monitoring.md:11]:
```suggestion
        <p>Security monitoring is available for packages in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
```

File: docs/security-monitoring.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
 </div>
 
 
 ### Configuring Monitoring Settings
-The default branch of every private package is automatically monitored if a composer.lock file is present.
+The default branch of every project is automatically monitored if a composer.lock file is present.
 Additional branches to be monitored can be selected on the package page.
 
-Security monitoring can be disabled for individual packages on the package security page or for all packages
+Security monitoring can be disabled for individual projects on the package security page or for all projects
```
Comment (naderman) [inline docs/security-monitoring.md:20]:
```suggestion
Security monitoring can be disabled for individual project packages on the package security page or for all projects
```

File: docs/security-monitoring.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
     <div class="callout warning">
-        <p>Security monitoring is available for packages in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
+        <p>Security monitoring is available for projects in your organization. To monitor your projects, you must add them to Private Packagist as packages, even if you do not intend to install them as a dependency.</p>
     </div>
 </div>
 
 
 ### Configuring Monitoring Settings
-The default branch of every private package is automatically monitored if a composer.lock file is present.
+The default branch of every project is automatically monitored if a composer.lock file is present.
```
Comment (naderman) [inline docs/security-monitoring.md:17]:
```suggestion
The default branch of every project is automatically monitored if a composer.lock file is present.
```

Comment (glaubinix) [reply docs/security-monitoring.md:17]: @naderman there are not suggested changes here. Did you miss something?

Comment (naderman) [reply docs/security-monitoring.md:17]: haha no i meant to undo this, and didn't realize that it's still a "comment" so I cannot just edit the content to be the original to undo the comment :D

**Pull request merged**